### DISCUSSION TITLE
Fresh Docker Install

### DISCUSSION THREAD

**Discussion by ghost** [AzuraCast/AzuraCast#5445] [Q&A] [answered]
Powered by AzuraCast
v0.16.1 Stable
Docker
PHP 8.1
Ubuntu 20.04LTS

/var/azuracast is empty except for the files below:
azuracast.env
docker-compose.yml
docsker.sh
.env

when I did an Ansible install I couldn't get SSL working but the directory structure was present
there has to be something I'm missing but I can't find it in the Docs, Discussions or Issues
what should be a 15 to 20 minute install has now taken 5 days

I found the path for the files
/var/lib/docker/volumes/azuracast_station_data/_data
nowhere in the documents did I find anything similar. Every path I saw was /var/azuracast
when were the documents updated last?

**Comment by ItsMitchh** [comment, marked as answer]
>/var/azuracast is empty except for the files below:

This is because we keep our docker container information inside the container itself, it's 'hidden' away in the docker/volumes. 90% of our users generally don't need to know where to find the files or data. 

>when were the documents updated last?

2 days ago.

**Reply by ghost** [reply]
well for the 10% of us who do need to know that info it should be available. This is just another reason I hate docker, I don't have time to spend days looking for the info I need.
